include/swift/Basic/CASOptions.h

```diff
@@ -80,6 +81,15 @@ class CASOptions final {
   llvm::hash_code getModuleScanningHashComponents() const {
     return getPCHHashComponents();
   }
+
+  /// Return corresponding clang::CASOptions for swift CASOptions.
```

Comment:
Nit: should this be `swift::CASOptions`?

Reply:
I just deleted. This doesn't provide much value and I directly inline the implementation to its only call site.